### ISSUE TITLE
Service tags extraction problem

### ISSUE THREAD

**Issue by rayrutjes** [tutumcloud/stackfiles#67]
I have share the following stack file: https://stackfiles.io/registry/5606bd1e31f4d50100cc9d1d and the service tags displays numbers from 1 to 52 instead of the only available service: gitlab-simple-runner

In the search results it also displays: 53 services

**Comment by MaximeHeckel** [comment]
Yes it a known issue, we still need to figure out why and how it happens, we think it might be syntax related. 
What happens if you delete it and repost it?
